# Question

This code snippet makes a fetch to get the first todo item from the JSON placeholder API. Or at least it tries to.

What is wrong with this code? 

Describe how async/await helps us avoid this mistake.

```js
fetch('https://jsonplaceholder.typicode.com/todos/1')
  .then(response => {
    if (!response.ok) throw new Error('Fetch failed!');
    response.json();
  })
  .then(json => console.log(json))
  .catch(err => console.log(err))
```

You can run this code of course, but try not to! Try to figure out the issue on your own.

# Answer
The issue with this code snippet is related to a common mistake when using `.then`. `.then` allows for chaining promises but as suggested, it needs a promise to be passed into the function for anything to be executed (imagine a function without the neccesary argument). In this code snippet, a promise is not being returned from the first `.then()`

# Teacher Feedback
3/3